quick_test.py

```diff
@@ -0,0 +1,13 @@
+import torch
```

Comment:
perhaps we can delete this file? looks like it is well tested in test_operators.py